draft-ietf-quic-recovery.md

```diff
@@ -1147,6 +1161,8 @@ acked_packet from sent_packets.
    OnPacketAckedCC(acked_packet):
      // Remove from bytes_in_flight.
      bytes_in_flight -= acked_packet.size
+     if (bytes_in_flight != 0):
+       probe_start_time = Now()
```

Comment:
I don't think it's necessary to check bytes_in_flight != 0 here?

Reply:
Yeah, you are right. I'm not sure what I was thinking when I added this. I will remove it.